src/ipa_pc/mod.rs

```diff
@@ -165,18 +149,18 @@ impl<G: AffineCurve, D: Digest> InnerProductArgPC<G, D> {
             let hiding_comm = proof.hiding_comm.unwrap();
             let rand = proof.rand.unwrap();
 
-            let hiding_challenge = Self::compute_random_oracle_challenge(
-                &to_bytes![combined_commitment, point, combined_v, hiding_comm].unwrap(),
-            );
+            fs_rng.absorb(&to_bytes![combined_commitment, point, combined_v, hiding_comm].unwrap());
```

Comment:
combined_commitment, combined_v were already absorbed and point was squeezed. Don't absorb